olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationClient.h

```diff
@@ -270,6 +268,14 @@ class AUTHENTICATION_API AuthenticationClient {
       const std::string& authentication_server_url = kHereAccountProductionUrl,
       size_t token_cache_limit = 100u);
 
+  /**
+   * @brief Creates the `AuthenticationClient` instance.
+   *
+   * @param settings The authentication settings that can be used to configure
+   * the `AuthenticationClient` instance.
+   */
+  AuthenticationClient(AuthenticationSettings settings);
```

Comment:
```suggestion
  explicit AuthenticationClient(AuthenticationSettings settings);
```
Better to create ctor as explicit to avoid implicit constructions from settings instance in the code.